mlos_bench/mlos_bench/environments/remote/remote_env.py

```diff
@@ -66,9 +66,10 @@ def __init__(self,
             "RemoteEnv requires a service that supports remote execution operations"
         self._remote_exec_service: SupportsRemoteExec = self._service
 
-        assert self._service is not None and isinstance(self._service, SupportsHostOps), \
-            "RemoteEnv requires a service that supports host operations"
-        self._host_service: SupportsHostOps = self._service
+        if self._wait_boot:
```

Comment:
Quick hack around the fact that `SshService` doesn't implement `SupportsHostOps` since there's no way to do `start_host` with it.

Might need a more complete fix in the future ... or now ...

Reply:
Skipping for now, but we should revisit, especially since there are currently no tests for that operation according to the code coverage results.